clients/docs/includes/configs/cloud/java.config

```diff
@@ -1,6 +1,9 @@
-# Kafka
+# Kafka client (producer/consumer/admin)
```

Comment:
If we do decide to keep the `Kafka producer` section (pending other comment), perhaps we can rename the headings as:

`# Kafka producer, consumer, and admin`
`# Kafka producer only`

Reply:
Sounds good, I'll update once we reach some consensus on the other discussion.

Reply:
Brainstorming out loud: maybe we tweak the headings to address the point "Stepping back, given that these are optimizations and not required, do they even belong in this file? Original intention was that these were just minimum set of required parameters to connect to a Kafka cluster. Should this PR go just to docs?"

For example, first heading could be:

`# Required connection settings for Kafka producer, consumer, and admin`

Second heading could be:

`# Recommended settings for Kafka producer`

That would more clearly communicate the differences.  Not just that they are for producer, but that they are optional (but recommended)...versus **required** for connectivity.

WDTY @ijuma @rspurgeon  ?

Reply:
This idea seems promising. What about `acks=all` though? That is required for not losing data, it's not about optimal performance. Where would you have that?